### PR TITLE
Debug vector add Windows CI failure.

### PR DESCRIPTION
**Pull request by sgundabo**
fixed missing \<CR\> in the kernel

### CODE REVIEW

**Review by junliume — APPROVED** [review]

**Review by apwojcik — APPROVED** [review]

File: test/gtest/vec_add.cpp
Comment (amberhassaan) [inline test/gtest/vec_add.cpp:117]: I hope you have learned by now that `uint` is not the correct nomenclature :).

File: test/gtest/vec_add.cpp
Comment (amberhassaan) [inline test/gtest/vec_add.cpp]: Several minor but important issues to address: 
* `vecadd_config` and `vec_size, threads_per_block` are duplicating the same info. Keep `vecadd_config` and get rid of the other two members.

File: test/gtest/vec_add.cpp
Comment (amberhassaan) [inline test/gtest/vec_add.cpp]: Do not convert `size_t` to `int`. That's a `narrowing` conversion. Either keep everything as `int or unsigned int` or keep everything as `size_t`.

**Review by amberhassaan — APPROVED** [review]
Please address the comments before merging.

**Comment by sgundabo** [comment]
@junliume Please merge this when you get a chance.

**Review by junliume — APPROVED** [review]

**Pull request merged**
